### ISSUE TITLE
Libunbound openssl error - 0.11.0.0 release version

### ISSUE THREAD

**Issue by dEBRUYNE-1** [monero-project/monero#2411]
A few users have been reporting these "errors" on Linux 64-bit (using the latest release, Helium Hydra):

>[1504816717] libunbound[25519:0] info: warning: unsupported algorithm for trust anchor . DS IN

>[1504816717] libunbound[25519:0] warning: trust anchor . has no supported algorithms, the anchor is ignored (check if you need to upgrade unbound and openssl)

As far as I can tell, it doesn't inhibit them from syncing or using the daemon. 

Source:

https://www.reddit.com/r/Monero/comments/6yprar/mandatory_upgrade_monero_01100_helium_hydra/dmpa8ab/

**Comment by moneromooo-monero** [comment]
This prevents DNSSEC verification, but it's apparently not done well in the first place anyway. This looks like an OpenSSL which was compiled with SHA1 (or a cmake bug which causes USE_SHA1 to be undefined when building unbound).

**Comment by RichAyotte** [comment]
Possibly some relevant information here.

https://www.unbound.net/pipermail/unbound-users/2016-January/004176.html

**Comment by anonimal** [comment]
(referencing #2133 for housekeeping)

**Comment by MattTwinkleToes** [comment]
I am afraid that it does see to stop us syncing or even accessing the wallet at all.  MY balance is 0.00000 at the moment.

```
./monerod 
2017-09-18 23:08:31.458	    7f7256275740	INFO 	global	src/daemon/main.cpp:279	Monero 'Helium Hydra' (v0.11.0.0-release)
2017-09-18 23:08:31.458	    7f7256275740	INFO 	global	src/daemon/protocol.h:55	Initializing cryptonote protocol...
2017-09-18 23:08:31.458	    7f7256275740	INFO 	global	src/daemon/protocol.h:60	Cryptonote protocol initialized OK
2017-09-18 23:08:31.458	    7f7256275740	INFO 	global	src/daemon/p2p.h:63	Initializing p2p server...
[1505776111] libunbound[2528:0] info: warning: unsupported algorithm for trust anchor . DS IN
[1505776111] libunbound[2528:0] warning: trust anchor . has no supported algorithms, the anchor is ignored (check if you need to upgrade unbound and openssl)
2017-09-18 23:08:33.208	    7f7256275740	INFO 	global	src/daemon/p2p.h:68	P2p server initialized OK
2017-09-18 23:08:33.208	    7f7256275740	INFO 	global	src/daemon/rpc.h:58	Initializing core rpc server...
2017-09-18 23:08:33.208	    7f7256275740	INFO 	global	contrib/epee/include/net/http_server_impl_base.h:70	Binding on 127.0.0.1:18081
2017-09-18 23:08:33.209	    7f7256275740	INFO 	global	src/daemon/rpc.h:63	Core rpc server initialized OK on port: 18081
2017-09-18 23:08:33.209	    7f7256275740	INFO 	global	src/daemon/core.h:73	Initializing core...
2017-09-18 23:08:33.210	    7f7256275740	INFO 	global	src/cryptonote_core/cryptonote_core.cpp:323	Loading blockchain from folder /home/matt/.bitmonero/lmdb ...
2017-09-18 23:08:33.406	    7f7256275740	ERROR	cn	src/cryptonote_basic/cryptonote_format_utils.cpp:102	Failed to parse transaction from blob
2017-09-18 23:08:33.406	    7f7256275740	ERROR	txpool	src/cryptonote_core/tx_pool.cpp:1020	Failed to parse tx from txpool
2017-09-18 23:08:33.406	    7f7256275740	ERROR	cn	src/cryptonote_core/cryptonote_core.cpp:409	Failed to initialize memory pool
2017-09-18 23:08:33.406	    7f7256275740	INFO 	global	src/daemon/rpc.h:90	Deinitializing rpc server...
2017-09-18 23:08:33.406	    7f7256275740	INFO 	global	src/daemon/p2p.h:90	Deinitializing p2p...
2017-09-18 23:08:33.406	    7f7256275740	INFO 	global	src/daemon/core.h:89	Deinitializing core...
2017-09-18 23:08:33.416	    7f7256275740	INFO 	global	src/daemon/protocol.h:77	Stopping cryptonote protocol...
2017-09-18 23:08:33.416	    7f7256275740	INFO 	global	src/daemon/protocol.h:81	Cryptonote protocol stopped successfully
```

**Comment by moneromooo-monero** [comment]
Can you file new bugs separately please, or this is going to be really confusing. One of the txes in the pool seems to be corrupt.

**Comment by moneromooo-monero** [comment]
Might be fixed by https://github.com/monero-project/monero/pull/2996

**Comment by moneromooo-monero** [comment]
Can someone who had that problem confirm whether this is fixed ?

**Comment by salim-b** [comment]
it's fixed for me (Ubuntu 16.04 LTS) :slightly_smiling_face:

**Comment by moneromooo-monero** [comment]
Thanks

+resolved
